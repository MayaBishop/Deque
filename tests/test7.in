#testing clearing on a deque of 1 and front and back commands
enqueue_front 1
clear
print
enqueue_front 1
enqueue_back 2
enqueue_front 3
front 1
front 3
print
back 3
back 2
