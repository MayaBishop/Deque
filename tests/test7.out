success
success
success
success
success
failure
success
3 1 2
2 1 3
failure
success
